ee/dtr/index.md

```diff
@@ -6,61 +6,41 @@ keywords: registry, repository, images
-cloud. And with it, you can store your Docker images securely, behind your
-firewall.
+* [Image and Job Management](image-and-job-management)
+* [Availability](availability)
+* [Efficiency](efficiency)
+* [Built-in Access Control](built-in-access-control)
+* [Security Scanning](security-scanning)
+* [Image Signing](image-signing)
 
-You can use DTR as part of your continuous integration, and continuous
-delivery processes to build, ship, and run your applications.
+## Image and Job Management
+ DTR offers a web-based user interface with which users can browse Docker images and [review repository events](/ee/dtr/user/audit-repository-events/). With the UI, in fact, users can even see which Dockerfile lines were used to produce an image and, if security scanning is enabled, a list of all of the software installed in that image. In addition, the web UI can be used to [review and audit jobs](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
 
-DTR has a web user interface that allows authorized users in your
-organization to browse Docker images and [review repository events](/ee/dtr/user/audit-repository-events/). It even allows you to see what Dockerfile
-lines were used to produce the image and, if security scanning is enabled, to
-see a list of all of the software installed in your images. Additionally, you can now [review and audit jobs on the web interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
+DTR can serve as a Continuous Integration and Continuous Delivery component, in the building, shipping, and running of applications.
 
 ## Availability
 
-DTR is highly available through the use of multiple replicas of all containers
-and metadata such that if a machine fails, DTR continues to operate and can be repaired.
+DTR is highly available through the use of multiple replicas of all containers and metadata. As such, DTR will continue to operate in the event of machine failure, thus allowing for repair.
 
 ## Efficiency
 
-DTR has the ability to [cache images closer to users](admin/configure/deploy-caches/index.md)
-to reduce the amount of bandwidth used when pulling Docker images.
+DTR is able to reduce the amount of bandwidth used when pulling Docker images by [caching images closer to users](admin/configure/deploy-caches/index.md). in addition, DTR can [clean up unreferenced manifests and layers](admin/configure/garbage-collection.md).
 
-DTR has the ability to [clean up unreferenced manifests and layers](admin/configure/garbage-collection.md).
+## Built-in Access Control
 
-## Built-in access control
+DTR uses the [same authentication mechanism](https://docs.docker.com/ee/ucp/#built-in-security-and-access-control) as the Universal Control Plane, with which users can be managed manually or synchronized from LDAP or Active Directory. DTR employs [Role Based Access Control](admin/manage-users/index.md) (RBAC), which allow the implementation of fine-grained access control policies for Docker images.
```

Comment:
"...Universal Control Plane (UCP)..."

"...Role-based..."

"...which allows..."

Reply:
how about:
"DTR uses [Role Based Access Control (RBAC)](admin/manage-users/index.md), which allows you to manage access manually or with LDAP or Active Directory."

If it's important to say that it's the same mechanism that UCP uses, maybe add "Like UCP, " to the beginning of the sentence?

Reply:
Feedback integrated. Thanks.